pkg/operator/configobserver/node/observe_latency_profile.go

```diff
@@ -56,42 +53,37 @@ func (l *latencyProfileObserver) observeLatencyProfile(
 	existingConfig map[string]interface{},
 ) (ret map[string]interface{}, errs []error) {
 
-	// traverse list of latencyConfigs to get all the config paths
-	configPaths := make([][]string, len(l.latencyConfigs))
-	for i, latencyConfig := range l.latencyConfigs {
-		configPaths[i] = latencyConfig.ConfigPath
-	}
-
 	defer func() {
 		// Prune the observed config so that it only contains fields specific to this observer
-		ret = configobserver.Pruned(ret, configPaths...)
+		ret = configobserver.Pruned(ret, l.usedConfigPaths...)
 	}()
 
+	for _, shouldSupressConfigUpdatesFn := range l.shouldSuppressConfigUpdatesFuncs {
```

Comment:
what is the reason why supressor resolution is before the cluster object resolution?

Reply:
Couldn't recall anything specific in mind for the reasoning behind using the suppressors before the cluster object resolution, apart from maybe customizability of the suppressor callers. The only point of concern was:
```go
if configNode.Spec.WorkerLatencyProfile == "" { ... }
```
from not hindering the flow of rejection from/to "" <-> Low profile(s). But that's still possible with moving the cluster object resolution above the suppressor and putting the if statement after calling the suppressors, so updated it that way.